src/editors/containers/VideoUploadEditor/hooks.js

```diff
@@ -0,0 +1,42 @@
+import * as requests from '../../data/redux/thunkActions/requests';
+
+export const uploadVideo = ({ dispatch, supportedFiles }) => {
+  const data = { files: [] };
+  supportedFiles.forEach((file) => {
+    data.files.push({
+      file_name: file.name,
+      content_type: file.type,
+    });
+  });
+  dispatch(requests.uploadVideo({
+    data,
+    onSuccess: (response) => {
+      const { files } = response.json();
+      Object.values(files).forEach((fileObj) => {
+        const fileName = fileObj.file_name;
+        const uploadUrl = fileObj.upload_url;
```

Comment:
FYI, you can do this, but I don't know which one is better:
```suggestion
        const { file_name: fileName, upload_url: uploadUrl } = fileObj;
```